packages/cli/snippets/oauth2-pkce.js

```diff
@@ -14,6 +14,8 @@ const authentication = {
         state: '{{bundle.inputData.state}}',
         redirect_uri: '{{bundle.inputData.redirect_uri}}',
         response_type: 'code',
+        code_challenge_method: 'S256', // Added for PKCE
+        code_challenge: '{{bundle.inputData.code_challenge}}', // Added for PKCE
```

Comment:
I don't think these are necessary. Our backend computes those values for us, and we don't read them from the config.